subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MavenBomResolveIntegrationTest.groovy

```diff
@@ -28,6 +28,7 @@ class MavenBomResolveIntegrationTest extends AbstractHttpDependencyResolutionTes
         resolve.prepare()
         settingsFile << """
             rootProject.name = 'testproject'
+            gradle.experimentalFeatures.enable('optionalDependencies')
```

Comment:
It would be nice to extract this into an `OptionalDependenciesTestFixtures.enable(File)` so that when we enable it by default in 5.0 we can search for usages.

Reply:
I'm not sure why you're commenting on this change, which is outdated by a later commit in this PR. I removed the ability to enable separate features via the DSL, since we decided to treat this as an all-or-nothing feature flag.